packages/client/vite.config.ts

```diff
@@ -6,11 +6,12 @@ import { isArray, mergeWith } from 'lodash'
 import path from 'path'
 import { defineConfig, loadEnv, UserConfig, UserConfigExport } from 'vite'
 import viteCompression from 'vite-plugin-compression'
-import { injectHtml } from 'vite-plugin-html'
+import { createHtmlPlugin } from 'vite-plugin-html'
 import PkgConfig from 'vite-plugin-package-config'
 
 import { getClientSetting } from './scripts/getClientSettings'
-import OptimizationPersist from './scripts/viteoptimizeplugin'
+
+// import OptimizationPersist from './scripts/viteoptimizeplugin'
```

Comment:
I think we can remove this line because `viteoptimizeplugin.js` file is deleted in this PR